### PR TITLE
change PROJECT_ROOT setting & SoftwareCompany -> Team

### PR DESCRIPTION
**Pull request by garylin2099**
1. Change PROJECT_ROOT  setting. Initial setting is designed for users who git clone the repo. As we metagpt is also a pypi package now, errors come up when users import it. This PR is a temporary fix to this problem. A complete solution will be to set PROJECT_ROOT by detecting if metagpt is git cloned or pip installed.
2. Rename SoftwareCompany to Team to represent a multi-agent entity of more general purpose.

中：
1. 修改 PROJECT_ROOT 设置。之前的设计假设用户会git clone仓库来使用MG。由于MG 现在也是一个 pypi 包，当用户导入它时会出现错误。这个 PR 是对这个问题的一个临时修复。完整的解决方案可能是通过检测 metagpt 是git clone的还是通过 pip 安装的来设置 PROJECT_ROOT
2. 将 SoftwareCompany 重命名为 Team，以代表一个更通用目的的多智能体实体

### CODE REVIEW

File: metagpt/team.py [hunk 1]
```diff
@@ -16,10 +16,10 @@
 from metagpt.utils.common import NoMoneyException
 
 
-class SoftwareCompany(BaseModel):
```
Comment (shenchucheng) [inline metagpt/team.py:19]: 是否有必要加个弃用申声明，而不是直接删除，类似

metagpt/software_company.py
```
import warnings
from metagpt.team import Team as SoftwareCompany

# 发出警告，指定将要被移除的版本号
warnings.warn("metagpt.software_company is deprecated and will be removed in version X.Y.Z. "
              "Please use metagpt.team instead. SoftwareCompany class is now named as Team.",
              DeprecationWarning, 2)
```

Comment (garylin2099) [reply metagpt/team.py]: Done

**Comment by stellaHSR** [comment]
LGTM

**Comment by shenchucheng** [comment]
LGTM

**Pull request merged**